data_model/src/isi.rs

```diff
@@ -143,43 +143,42 @@ pub mod model {
         /// `Upgrade` variant.
         Upgrade(UpgradeBox),
     }
+}
+
+mod shared {
+    // NOTE: instructions in this module don't have to be made opaque with `model!`
+    // because they are never shared between client and server(http)/host(wasm)
```

Comment:
So the module name is `shared` but things inside it are never shared...

Reply:
And you are abusing the fact the `#[model]` macro doesn't process nested modules. I'd better move that module out of `model` module

Reply:
> And you are abusing the fact the #[model] macro doesn't process nested modules. I'd better move that module out of model module

I am not abusing anything, this is not inside `model` module

> So the module name is shared but things inside it are never shared...

I will rename the module to `transparent`

Reply:
> I am not abusing anything, this is not inside model module

Hm, maybe I was misleaded by GitHub highlightning...